db/migrate/20220110090402_create_movies.rb

```diff
@@ -0,0 +1,11 @@
+class CreateMovies < ActiveRecord::Migration[7.0]
+  def change
+    create_table :movies do |t|
+      t.string :description
+      t.string :title
+      t.float :duration
```

Comment:
Duration in float? You want to store for instance `1,30` value? What that means? 1 h and 30 minutes or 1 hour and 18 minutes (30% of hour)?

Consider to store duration in minutes so it might be integer. It should be easier to compute later.